Cargo.toml

```diff
@@ -63,6 +63,10 @@ actix-ws = "0.3.0"
 # Utilities for futures which we use for the dev server
 futures-util = "0.3.31"
 
+# Logging crates
+tracing = "0.1.41"
+tracing-subscriber = { version = "0.3.19", features = ["env-filter", "std"] }
```

Comment:
```suggestion
tracing-subscriber = { version = "0.3.19", features = ["env-filter"] }
```

The [`std` feature is enabled by default](https://github.com/tokio-rs/tracing/blob/311c31321657ba657ccf63e10bc92a45a079fa2b/tracing-subscriber/Cargo.toml#L27), so we don't need this!